core/protos/o2control.proto

```diff
@@ -429,6 +431,7 @@ message WorkflowTemplateInfo {
     string template = 2;
     string revision = 3;
     map<string, VarSpecMessage> varSpecMap = 4;
+    string description = 5;
```

Comment:
Oooooook, now I see what you did. You added a description field to any and all roles, not just to the root role as a way to input an environment description. This is not quite what I expected but on the other hand it might not hurt. Let's discuss this tomorrow while looking at the code.